app/controllers/main_controller.rb

```diff
@@ -27,4 +29,14 @@ def dashboard
     @statuses = Teacher.validated.group(:status).order(count_all: :desc).count
     @schools = School.validated.order(teachers_count: :desc).limit(20)
   end
+
+  private
+  def mapped_application_status(status)
+    status_map = {
+      "info_needed" => "requested to be updated",
+      "not_reviewed" => "not reviewed"
+    }
```

Comment:
```suggestion
    }.with_indifferent_access
```

This allows a hash to treat symbols and strings as equivalent keys.